### DISCUSSION TITLE
SMR and ASMB Token missing

### DISCUSSION THREAD

**Discussion by Timo37K** [iotaledger/firefly#7720] [Help & Support]
Hello everyone, 

It's been over a year since I last opened my firefly wallet. 
When I opened it, I noticed that my ASMB tokens were no longer there, just the SMRs. Then I decided to update my wallet (from version 1.4 to 2.1) and there was nothing. It's impossible to get my hands on the tokens I staked more than a year ago. 

I've checked forums and the like, and some people are talking about tangle, but I don't quite understand... I also have the impression that the ASMB token is no longer part of the network. I can't find any help to find my tokens.

Could you help me find my tokens and tell me what's new in IOTA? 

Many thanks in advance.

**Comment by begonaalvarezd** [comment]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.3 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, but the new iota airdrop is on the new app and the shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues
